dequelabs/cauldron
# test: convert icon tests to react testing library

Issue https://github.com/dequelabs/cauldron/issues/1159

This PR switches the `Icon` tests to React Testing Library.

## Validation steps
- Run `yarn test` to see the tests pass
- Verify that the tests are converted properly
